config/i18n-tasks.yml

```diff
@@ -18,7 +18,8 @@ data:
     ## Default:
     # - config/locales/%{locale}.yml
     ## More files:
-    # - config/locales/**/*.%{locale}.yml
+    - config/locales/**/*-%{locale}.yml
```

Comment:
Our locale files are: `config/locales/controller/controller-locale.yml`, so `i18n-tasks` previously wasn't picking them up by the default of only looking at `config/locales/%{locale}.yml`.

Reply:
Nice catch!